### DISCUSSION TITLE
Toggling column colours

### DISCUSSION THREAD

**Discussion by ivanleong** [patrykandpatrick/vico#810] [Questions]
### Question

I have a series of stacked columns. is it possible to tap a single column so that the rest of the columns appear dimmed? Or manipulate the chart so that we have 2 sets of dimmed and normal colours for selected x value.

![Screenshot 2024-07-23 at 23 03 58](https://github.com/user-attachments/assets/674f1f08-2796-4ee6-aee3-56da0d8180da)
![Screenshot 2024-07-23 at 23 04 05](https://github.com/user-attachments/assets/b8874d20-4723-4580-abb6-1e838da1dada)

### Vico version(s)

2.0.0-alpha.21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hi @ivanleong, please see #769.
